src/main/java/exception/DukeException.java

```diff
@@ -0,0 +1,30 @@
+package exception;
```

Comment:
One small suggestions: maybe you could add more sub-exception of DukeException for better organization : )